README.md

```diff
@@ -17,7 +17,7 @@ Written in Rust and distributed in multiple languages.
 
 This repository maintains the source code and release process for these projects:
 
--   [Slang Rust Crate + CLI](https://nomicfoundation.github.io/slang/user-guide/cargo-crate/)
```

Comment:
Just to add a bit more context on why I made the change:
- rustc uses "crate" as the "rusty" name for the single compilation unit
- Cargo as a package manager and the build system deals with "packages" (e.g. `-p` flag), that can contain multiple "crates" in the sense above (the lib target is the primary "crate" that is consumed downstream, but other targets like build script, binaries, integration tests etc. are compiled separately and thus are technically different rustc "crates")
- crates.io adds to the confusion by using "crate" name to mean a Cargo `.crate`-suffixed package artifact (as above, that can contain different targets, thus rustc "crates") uploaded to the registry

So, I'd consider "crate" to be a bit loaded and sometimes confusing.

I changed Cargo->Rust to make it a bit more about the language rather than the build system; In my experience "Cargo crate" as a term (albeit still correct wrt `.crate` package artifact and crates.io notion of a "crate") is definitely not as popular as "npm package" in comparison and it felt weird reading this on the doc page.

In this specific case/line, the usage might be slightly more correct since it's "a library crate + CLI-the-binary (packaged together)", but elsewhere I'd probably mildly prefer "Rust package" whenever we refer to what we ship and we want to emphasize Rust-the-language there (to avoid slightly loaded "crate") or "Cargo package" over "Cargo crate" in general when Cargo is the emphasis (these can be considered the same but I feel like "package" is more popular when coupled with "Cargo"; "crates" I would say is more often referred to separately)

In any case, really not important and truly nitpicking but wanted to give some context on the change!